Formula/porechop.rb

```diff
@@ -15,6 +15,8 @@ class Porechop < Formula
   depends_on "python"
 
   def install
+    xy = Language::Python.major_minor_version "python3"
+    ENV.prepend_create_path "PYTHONPATH", libexec/"lib/python#{xy}/site-packages"
```

Comment:
No need for this block. Please remove this block.
Please add `revision 1` between `sha256` and `head`.